### PR TITLE
Use r-base version as package version, pinning ensures right requirements

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dbast** [comment]
@conda-forge/r Can you please have a look. 

This requires also fixing/changing https://github.com/conda-forge/r-recommended-feedstock in the same way.


This can also updated in the same way https://github.com/conda-forge/r-essentials-feedstock/pull/4

**Comment by dbast** [comment]
Hah. That even installs `r-recommended` from defaults and thus builds fine, without https://github.com/conda-forge/r-recommended-feedstock/pull/6

**Comment by rvalieris** [comment]
yep same thing here, should be good to merge now that r-recommended is fixed.
also rerender to fix the maintainers

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Comment by dbast** [comment]
Was already rerendered locally.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,9 +29,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - johanneskoester
-    - bgruening
-    - daler
-    - jdblischak
-    - cbrueffer
-    - dbast
+    - conda-forge/r
```
Comment (isuruf) [inline recipe/meta.yaml:32]: This makes it recursive. You need the members explicitly listed here.

Comment (dbast) [reply recipe/meta.yaml:32]: Oh, right. Will fix.

**Pull request merged**